Logger/Logger.cpp

```diff
@@ -0,0 +1,26 @@
+#include <iostream>
+#include <fstream>
+
+#include "Logger.hpp"
+
+void Logger::log(std::string msg)
```

Comment:
const std::string&

const - bo nie zmieniamy w metodzie argumentu msg
& - to przekazywanie przez referencje, doczytacie co to